angular2-jwt.ts

```diff
@@ -15,46 +15,62 @@ export interface IAuthConfig {
   noTokenScheme?:boolean;
 }
 
+// This interface should be used in the ctor of AuthHttp.
+export interface IAuthConfigOptional {
+    headerName?: string;
+    headerPrefix?: string;
+    tokenName?: string;
+    tokenGetter?: any;
+    noJwtError?: boolean;
+    globalHeaders?: Array<Object>;
+    noTokenScheme?: boolean;
+}
+
+class AuthConfigConsts {
```

Comment:
How would you do it then?
I think it is more maintainable, if we have them under one namespace.